lab_2_keywords_cooccurrence/main.py

```diff
@@ -55,7 +92,21 @@ def calculate_word_degrees(candidate_keyword_phrases: KeyPhrases,
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (check_list(candidate_keyword_phrases, tuple) and check_list(content_words, str)):
+        return None
+    word_degrees_dict = {}
+    for phrase in candidate_keyword_phrases:
+        for word in phrase:
+            if word in word_degrees_dict:
+                if word in content_words:
```

Comment:
You can take that `if` out of this statement and out of `else` - this is common part.